test/libsolidity/semanticTests/errors/error_emit_from_module_via_member_access.sol

```diff
@@ -0,0 +1,31 @@
+==== Source: A.sol ====
+error ErrorA(string msg);
+==== Source: B.sol ====
+import * as A from "A.sol";
+
+error ErrorB(string msg);
+
+contract BContract {
+    error ErrorB(string msg);
+}
+==== Source: C.sol ====
+import * as B from "B.sol";
+
+contract CContract {
+    function error1() external {
+        revert B.ErrorB("B error");
+    }
+
+    function error2() external {
+        revert B.BContract.ErrorB("B.BContract error");
+    }
+
+    function error3() external {
+        revert B.A.ErrorA("B.A error");
+    }
+}
+
+// ----
+// error1() -> FAILURE, hex"a5f9ec67", hex"0000000000000000000000000000000000000000000000000000000000000020", hex"0000000000000000000000000000000000000000000000000000000000000007", hex"42206572726f7200000000000000000000000000000000000000000000000000"
```

Comment:
The way isoltest formats the output by default is not always the best. For errors it's often unreadable and you have to update it by hand to make it sensible:

```suggestion
// error1() -> FAILURE, hex"a5f9ec67", 0x20, 7, "B error"
```